dll/cpl/appwiz/createlink.c

```diff
@@ -176,6 +177,23 @@ BOOL IsInternetLocation(LPCWSTR pszLocation)
     return (PathIsURLW(pszLocation) || wcsstr(pszLocation, L"www.") == pszLocation);
 }
 
+void
+DoConvertNameForFileSystem(LPWSTR szName)
```

Comment:
What is the point of this function?

Reply:
OK.

Reply:
Actually it would be better to use the more complete `PathCleanupSpec()` function (that only takes UNICODE characters by the way), we have it implemented, and is documented:
https://docs.microsoft.com/en-us/windows/win32/api/shlobj_core/nf-shlobj_core-pathcleanupspec

Reply:
No, I can't agree. Windows actually deletes a colon of a drive label in this situation.

Reply:
Added a comment.

Reply:
https://bugs.winehq.org/show_bug.cgi?id=48532